content/reference/dos-mitigation.md

```diff
@@ -131,6 +133,8 @@ scope](https://github.com/libp2p/go-libp2p/blob/v0.22.0/p2p/host/resource-manage
 
 In rust-libp2p you can tune this with `ConnectionLimits` as explained above.
 
+Similarly js-libp2p users can adjust the [connection limits](https://github.com/libp2p/js-libp2p/blob/master/doc/LIMITS.md#connection-limits) as explained in the js-libp2p docs.
```

Comment:
The current connection limits doc doesn't discuss transient connections.  I left a comment in https://github.com/libp2p/js-libp2p/pull/1421 about covering this.

Reply:
I've mentioned the specific setting users can tweak - hopefully that's clearer